packages/image_picker/image_picker_for_web/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-## NEXT
+## 3.1.0
 
+* Add limit parameter to `pickMultiImage` and `pickMultipleMedia` to ios and Android.
```

Comment:
The web package shouldn't have any changes, since the `limit` parameter should only be added to the `MultiImagePickerOptions`, right?

Reply:
Right, I removed changes from web, linux, macos and windows packages. I only left dependency_overrides, to verify if changes to other packages would not mess things up.